src/App.css

```diff
@@ -1,38 +1,16 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
+.background {
   min-height: 100vh;
+  background-color: #d3f6f9;
```

Comment:
may be nice to add a comment so anyone coming in will know what color this is

Reply:
I added a comment for the color in the next push